config/locales/en.yml

```diff
@@ -931,9 +931,9 @@ en:
           brownfield: "Brownfield Land"
           cemetery: "Cemetery"
           commercial: "Commercial Area"
-          conservation: "Conservation"
-          construction: "Construction"
-          farm: "Farm"
+          conservation: "Conservation Area"
+          construction: "Construction Area"
+          farm: "Farmland"
```

Comment:
This key is deprecated and should be removed rather than changed. There's only a few instances of it remaining in the database in 2022 anyway.